core/src/test/resources/sql/join.iq

```diff
@@ -1004,4 +1004,30 @@ ON t1.a IS NOT DISTINCT FROM t2.a;
 
 !ok
 
+### [CALCITE-4915] Query with unqualified common column and NATURAL JOIN fails
```

Comment:
I assume you have validated the correctness of this output using some other means.

Reply:
@alex-plekhanov if you can answer these two questions maybe we can resolve this issue?

Reply:
Sorry for delay with answer. I've tested this query with PG, result is the same, except ordering. I've added ORDER BY now, to get more predictable results.
Yes, we can close CALCITE-4915